src/_common/filters/date.ts

```diff
@@ -20,11 +20,11 @@ const ShortcutFormats = {
 	shortTime: 'h:mm a',
 };
 
-export function date(now: Date | string | number, formatTo?: string) {
+export function date(now: Date | number, formatTo: string = ShortcutFormats.medium) {
```

Comment:
No need to specify string for formatTo, it'll infer it from the type of "medium"